structuralcodes/materials/constitutive_laws/_bilinearcompression.py

```diff
@@ -51,9 +51,13 @@ def get_stress(
         # Compute stress
         # If it is a scalar
         if np.isscalar(eps):
-            sig = 0
-            if self._fc / self._E <= eps <= 0:
-                sig = self._E * eps
+            sig = self._E * eps
+            if eps > 0:
+                return 0
+            if eps < self._eps_cu:
+                return 0
+            if sig < self._fc:
+                return self._fc
```

Comment:
I don't know if this makes sense to you, but I would suggest something like the following, makes sense?

```suggestion
            if eps > 0 or eps < self._eps_cu:
                return 0
            sig = self._E * eps
            return sig if sig > self._fc else self._fc
```

Reply:
sure, is this fine with you?

```python
            if eps > 0 or eps < self._eps_cu:
               return 0
            return max(self._E * eps, self._fc)
```

Reply:
Excellent!